### ISSUE TITLE
0.6.1-1 deb install does not complete

### ISSUE THREAD

**Issue by davidenglishmusic** [ysbaddaden/prax.cr#49]
I am running Ubuntu 16.10 and during the install, it returns an error:
```
Setting up prax (0.6.1-1) ...
dpkg: error processing package prax (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 prax
E: Sub-process /usr/bin/dpkg returned an error code (1)
```

0.6.0-1 installs correctly.

**Comment by halida** [comment]
Same issue here, I don't know how to view the error message, but I think it may relative with: https://github.com/ysbaddaden/prax.cr/pull/33

**Comment by ysbaddaden** [comment]
Closing. #33 is fixed and 0.6.1 is an old release.
